packages/cli-lib/github.js

```diff
@@ -19,17 +20,26 @@ const GITHUB_AUTH_HEADERS = {
  * @param {String} repoName - name of the github repository
  * @returns {Buffer|Null} Zip data buffer
  */
-async function fetchJsonFromRepository(repoName, filePath) {
+async function fetchJsonFromRepository(
```

Comment:
This does need to be `async`, since we need to `await` the get request in the function ([link](https://github.com/HubSpot/hubspot-cli/blob/120f24a8aab3aaf08633bda45f1b66f584b8cde5/packages/cli-lib/github.js#L32)). 

I actually discovered a bug with this. We were originally simply returning the request without an `await`, and so we were never hitting the `catch` block, so originally, I was unable to make a custom error message.